misc/config_tools/upgrade_scenario_xml

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+function usage() {
+	echo "Usage: $0 <scenario XML> [<version of the given scenario XML>]"
+	echo "Upgrade the given scenario XML to align with the current version of ACRN."
+	echo ""
+	echo "Example: $0 logical_partition.xml v2.3"
```

Comment:
Is there any existing version info on the XML file that can help the user determine what version his XML file is from?

Reply:
Why do you need to know the old XML file ACRN version? And no, there's no indication in the XML file which version of ACRN (or the tool) was used to create it.

Reply:
The version of the old XML is essential as it decides the set of XML transformations to be applied.

For example, assume in v2.4 we added `hv/FEATURES/PSRAM` and in v2.5 we rename it to `hv/FEATURES/SSRAM`.  There will be two transformations (in sequence):

1. Adding a `PSRAM` node under `hv/FEATURES` (if it does not exist in the old XML).
1. Renaming `hv/FEATURES/PSRAM` to `hv/FEATURES/SSRAM`.

If a given XML is already v2.5-compliant, reapplying the transformations will introduce another `hv/FEATURE/SSRAM` node, leading to schema validation failure.